src/sysen1/pword.2664

```diff
@@ -9826,7 +9850,11 @@ bltspc:	syscal sstatu,[val x ? val x ? val x ? val x ? val x
 	       blt x,spcend-1	   ;  all of them to the bitter end
 	       setom dm.flg
 	       ret ]
-	.lose		           ; Unknown machine!
+	;; unknown machine, default to MD
```

Comment:
Why default to MD?

Reply:
As good as any? If you want specific settings, you will have to edit the code anyway, but if you don't, then MD is OK? The main goal is to be able to set a new machine name in CONFIG with a minimum of other edits (TTYTYP and H3TEXT seems to suffice).

Reply:
Historical reasons.  DB used MD as the default (maybe Bawden's choice?), so I kept that for new machines.